samples/csharp/MicrosoftEduImportFromGoogle/Import.cs

```diff
@@ -0,0 +1,324 @@
+            return graphServiceClient.Education.Classes
+                    .GetAsync()
+                    .Result.Value.ToList();
+        }
+
+        public async Task<List<string>> MapAndCreateAssignments(CourseWork[] courseWorks, string classId, Export exporterInstance)
+        {
+			Console.WriteLine("* Importing coursework from Google Classroom into Microsoft Teams...");
+			List<string> assignmentsCreated = new List<string>();
+            foreach(var courseWork in courseWorks)
+            {
+                var createdAssignment = await CreateAssignmentAsync(classId,
+                    new EducationAssignment
+                    {
+                        DisplayName = courseWork.Title,
+                        Instructions = new EducationItemBody { Content = courseWork.Description },
+                        DueDateTime = DateTime.Now.AddDays(7),//revisit
+                    }
+                );
+                assignmentsCreated.Add(createdAssignment.DisplayName);
+                if (courseWork.Materials?.Any() == true)
+                {
+                    await MapAndCreateResources(courseWork.Materials, createdAssignment, exporterInstance);
+                }
+            }
+            return assignmentsCreated;
+        }
+
+
+        public async Task<List<string>> MapAndCreateModules(CourseWorkMaterials[] courseWorkMaterials, string classId, Export exporterInstance)
+        {
+            Console.WriteLine("* Importing coursework materials from Google Classroom into Microsoft Teams classwork...");
+            List<string> modulesCreated = new List<string>();
+            foreach (var courseWork in courseWorkMaterials)
+            {
+                var createdModule = await CreateModuleAsync(classId,
+                    new EducationModule
+                    {
+                        DisplayName = courseWork.Title,
+                        Description = courseWork.Description
+                    }
+                );
+                modulesCreated.Add(createdModule.DisplayName);
+
+                if (courseWork.Materials.Any() == true)
+                {
+                    await MapAndCreateModuleResources(courseWork.Materials,classId, createdModule, exporterInstance);
+                }
+            }
+            return modulesCreated;
+        }
+
+        private async Task MapAndCreateResources(List<Material> materials, EducationAssignment createdAssignment, Export exporterInstance)
+        {
+            foreach (var material in materials)
+            {
+                string fileName = null;
+                Byte[] fileAsByteArray = new byte[10000];
+                if (material.DriveFile != null)
+                {
+                    var sourceFileMetadata = await exporterInstance.GetGoogleDriveFileMetadata(material.DriveFile.DriveFile.Id);
+                    if (sourceFileMetadata["mimeType"].Contains("drawing")) // Skip Google drawing resources, not supported in Microsoft
```

Comment:
not needed since this validation is only for Google materials where there is a DriveFile which is not the case for forms.
forms resources are not imported by default